site/viamguides/src/emily-door-lock/emily-door-lock.md

```diff
@@ -0,0 +1,210 @@
+author: Joyce
+id: emily-door-lock
+summary: Monitor door lock position using an infrared sensor
+categories: Getting-Started, Developer
+environments: web
+status: Published
+feedback link: https://github.com/viam-devrel/viamcodelabs/issues
+tags: Getting Started, Developer
+
+# Detect door lock status with an infrared sensor
+
+<!-- ------------------------ -->
+
+## Overview
+
+**Is my door locked?** If you leave your home, and you're unsure whether you locked your front door, this is a smarter way to check whether your front door is locked or unlocked.
+
+In this codelab, you'll learn how to use an infrared sensor to see whether a door is locked or unlocked in realtime.
+
+### What You’ll Build
+
+- A Viam-powered internet-connected device that tells you whether your door is locked
+
+  ![IR sensor mounted next to door lock](assets/hero.png)
+
+### Prerequisites
+
+- A computer with MacOS, Windows, or Linux to flash your Raspberry Pi and configure the device's components using the Viam app
+- Hardware and supplies requirements
+
+  - 1 - [ESP32 development board](https://a.co/d/fNsT6d7) - check the Viam docs for [specific resource requirements](https://docs.viam.com/installation/#compatibility)
```

Comment:
This linked board only has 4MB of flash, which is half of the required amount. I've updated the link to one that meets the requirements and linked to the new docs page that lists them. 

```suggestion
  - 1 - [ESP32 development board](https://a.co/d/0liXUKP) - check the Viam docs for [specific resource requirements](https://docs.viam.com/operate/get-started/setup-micro/)
```